web/styles/settings.css

```diff
@@ -716,6 +716,11 @@ input[type="checkbox"] {
 
     .emoji_name_input {
         margin-top: 10px;
+
+        & label{
+            display: table;
+        }
+
```

Comment:
There are better ways to do it, but the fact that is just one property change, I am sure if other solutions are worth it.

Can you fix lining problems by running `./tools/lint --fix`?